esp8266/wifistation.ino

```diff
@@ -202,32 +202,70 @@ exec_cmd(char *cmd, size_t len)
 
 	switch (lcmd[2]) {
 	case 'd': {
-		char *host, *ohost;
+		char *host, *ohost, *bookmark;
 		uint16_t port;
 		int chars;
+		int index;
 
-		/* ATDT: dial a host */
-		if (len < 5 || lcmd[3] != 't')
+		if (len < 5)
 			goto error;
 
-		host = ohost = (char *)malloc(len);
-		if (host == NULL)
-			goto error;
-		host[0] = '\0';
-
-		if (sscanf(lcmd, "atdt%[^:]:%d%n", host, &port, &chars) == 2 &&
-		    chars > 0)
-			/* matched host:port */
-			;
-		else if (sscanf(lcmd, "atdt%[^:]%n", host, &chars) == 1 &&
-		    chars > 0)
-		    	/* host without port */
-			port = 23;
-		else {
-			errstr = strdup("invalid hostname");
+		switch (lcmd[3]) {
+		/* ATDT: dial a host */
+		case 't':
+			host = ohost = (char *)malloc(len);
+			if (host == NULL)
+				goto error;
+			host[0] = '\0';
+			if (sscanf(lcmd, "atdt%[^:]:%d%n", host, &port, &chars) == 2 &&
```

Comment:
This and any other re-indented lines need to be re-wrapped at 80 chars.

Reply:
Is your tabstop set to 8?

Reply:
Yeah, basically OpenBSD's [`style(9)`](http://man.openbsd.org/style.9)